sconscontrib/SCons/Tool/pandoc/__init__.py

```diff
@@ -0,0 +1,539 @@
+# MIT License
+#
+# Copyright 2021 The SCons Foundation
+# Copyright 2016-2021 Keith F Prussing
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY
+# KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+# LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+__doc__ = """SCons.Tool.pandoc
+
+The Tool specific initialization for the Pandoc document conversion
+command line tool.
+
+There normally shouldn't be any need to import this module directly.  It
+will usually be imported through the generic SCons.Tool.Tool() selection
+method.
+
+"""
+
+import SCons.Action
+import SCons.Builder
+import SCons.Scanner
+import SCons.Util
+try:
+    from SCons.Warnings import SConsWarning as SConsWarning
+except ImportError:
+    from SCons.Warnings import Warning as SConsWarning
+
+import argparse
+import logging
+import os
+import re
+import shlex
+import subprocess
+import sys
+
+try:
+    import importlib.metadata as metadata
+except ImportError:
+    # Work around the fact that mypy has a bug
+    #
+    # -   https://github.com/python/mypy/issues/1153
+    import importlib_metadata as metadata  # type: ignore
+
+if sys.version_info < (3, 6):
```

Comment:
Can you stick a 
`#TODO: Remove once SCons drops Python 3.6`
Here?
We expect to do that soon?